src/abstractions/software/assembler/index.test.js

```diff
@@ -0,0 +1,12 @@
+import Assembler from './index'
+import {
+  SYMBOL_LESS_ASSEMBLY_CODE,
+  SYMBOL_LESS_MACHINE_CODE
```

Comment:
There are more than one symbol less assembly codes that come with the nand2tetris book. Can we include all them in the test. It will make the test more robust.